generate.go

```diff
@@ -9,9 +9,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/libgen"
```

Comment:
@wking , @mrunalp . Sounds good. I will modify the code structure as @wking  suggested.
@wking , in your proposal, there are two `generate.go` - one is under `cmd/ocitools`, another is under `generate`. What is their difference? The same problem happens to `validate.go`.
I think all the files under `generate`  and `validate` should be libraries, right?